packages/core/content-manager/admin/src/hooks/useDocumentActions.ts

```diff
@@ -533,6 +583,7 @@ const useDocumentActions: UseDocumentActions = () => {
     discard,
     getDocument,
     publish,
+    bulkPublish,
```

Comment:
Lets call it `publishMany` I think it's better name imo

Reply:
👍🏻 even though that wont match the endpoint on the BE? Or you mean to update
it there too?

Reply:
i think its fine it doesnt match the endpoint, it's clearer name for the hook's API 😄